hack/prepare-operator-extension.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -eu
```

Comment:
Nit: Should we use the same bash flags as in most other scripts?
```suggestion
set -euo pipefail
```